shoop/campaigns/models/catalog_filters.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from polymorphic.models import PolymorphicModel
+
+from shoop.core.models import Category, Product, ProductType
+
+
+class CatalogFilter(PolymorphicModel):
+    model = None
+
+    identifier = "base_catalog_filter"
+    name = _("Base Catalog Filter")
+
+    active = models.BooleanField(default=True, verbose_name=_("active"))
+
+    def filter_queryset(self, queryset):
+        raise NotImplementedError("Subclasses should implement `filter_queryset`")
+
+    @property
+    def description(self):
+        extra = "(s)" if self.model else ""
+        return _("Define the required %s%s for this campaign." % (self.name.lower(), extra))
```

Comment:
No, it's still untranslatable. Better to just overwrite this description for each of the subclasses to make this easier for translators. Then this would be just e.g. `_("Define the required product types for this campaign")` for the `ProductTypeFilter` and so on for all of them.